1_fetch/src/get_nwis_data.R

```diff
@@ -1,38 +1,61 @@
-
-download_nwis_data <- function(site_nums = c("01427207", "01432160", "01435000", "01436690", "01466500")){
-  
-  # create the file names that are needed for download_nwis_site_data
-  # tempdir() creates a temporary directory that is wiped out when you start a new R session; 
-  # replace tempdir() with "1_fetch/out" or another desired folder if you want to retain the download
-  download_files <- file.path(tempdir(), paste0('nwis_', site_nums, '_data.csv'))
-  data_out <- data.frame()
-  # loop through files to download 
-  for (download_file in download_files){
-    download_nwis_site_data(download_file, parameterCd = '00010')
-    # read the downloaded data and append it to the existing data.frame
-    these_data <- read_csv(download_file, col_types = 'ccTdcc')
-    data_out <- bind_rows(data_out, these_data)
-  }
-  return(data_out)
+#' Download NWIS data
+#' 
+#' Download data from National Water Information System 
+#' 
+#' @param site_no chr, valid 8-digit USGS NWIS gage number. Include leading zero.
+#' @param pathout chr, file path for output `csv` file. Path only; do not include file name.
+#' 
+download_nwis_data <- function(site_no, pathout = '1_fetch/out/'){
+  download_file <- paste(pathout, "/nwis_", site_no, ".csv", sep = "")
```

Comment:
I might have missed this last time but I think you are throwing in an extra `/` here. You should be able to change the file name prefix to be `nwis` instead of `/nwis'

Reply:
I'll also note that this would be just a matter of preference so ok to leave as is but you could for simplicity pass in the file path directly to the `download_nwis_site_data` function rather than call this function which just assembles the file path and then calls the `download_nwis_site_data` function. So instead you would just specify the file path right in your `_targets.R` file and call the `download_nwis_site_data` function directly.